docs/nodes/nodes-library/core-nodes/SSH/README.md

```diff
@@ -0,0 +1,68 @@
+---
+permalink: /nodes/n8n-nodes-base.ssh
+description: Learn how to use the SSH node in n8n
+---
+
+# SSH
+
+The SSH node is useful for executing commands via the Secure Shell Protocol.
```

Comment:
Do you think it might be a good idea to give a few cases as an example here? This will help non-technical users know how they can use this node.